source/_integrations/igloohome.markdown

```
@@ -0,0 +1,77 @@
+
+- Ensure that your subscription is still active.
+
+### Battery levels not updating
+
+- If you are using a bridge, ensure that it is linked to the correct devices.
+- If you do not have a bridge. Sync the device with the igloohome mobile app, then reload the config entry associated to the device.
+
+### Authentication problems
+
+- Verify that your [API credentials](https://access.igloocompany.co/api-access) are correctly filled in.
+- Ensure your API credentials have not expired or been revoked.
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)